common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1790,6 +1790,10 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVEALIAS("hive.alias", "", ""),
     HIVEMAPSIDEAGGREGATE("hive.map.aggr", true, "Whether to use map-side aggregation in Hive Group By queries"),
     HIVEGROUPBYSKEW("hive.groupby.skewindata", false, "Whether there is skew in data to optimize group by queries"),
+
+    HIVE_ENABLE_COMBINER_FOR_GROUP_BY("hive.enable.combiner.for.groupby", true,
+        "Whether to enable tez combiner to aggregate the records after sorting is done"),
```

Comment:
Maybe clarify it is only used for map side aggregation? Any case this would not be beneficial?

Reply:
done